src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -21,7 +26,7 @@ public void OnAttachToCell(PlacedOrganelle organelle)
     {
         this.organelle = organelle;
         organellePos = Hex.AxialToCartesian(organelle.Position);
-
+        defaultVisualPos = new Vector3(0, 0, -1);
```

Comment:
Whats the difference?

Reply:
It's a proper constant instead of guessing that -1 in Z axis is forward.
I've learned not to trust values like this by changing graphics engines a bunch of times (and definitely the forward direction is *not* standard)

Reply:
Its default as ive defined it, it could by any pf the 4 Directions.